.github/workflows/build.yml

```diff
@@ -0,0 +1,69 @@
+name: Build struct2tensor wheels
+
+on:
+  pull_request:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Build package
+        id: build-package
+        uses: ./.github/reusable-build
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Retrieve wheels
+        uses: actions/download-artifact@v4
+        with:
+          merge-multiple: true
+          path: wheels
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: List and check wheels
+        shell: bash
+        run: |
+          pip install twine pkginfo>=1.10.0
```

Comment:
Is `pkginfo` used for twine?

Reply:
Yeah, annoyingly we need to manually pin this transitive dependency until [this is fixed](https://github.com/pypa/twine/issues/1070).